packages/vscode-pyright/package.json

```diff
@@ -1763,6 +1763,18 @@
                         }
                     }
                 },
+                "basedpyright.analysis.fileEnumerationMinimumFiles": {
+                    "type": "integer",
+                    "default": 50,
+                    "description": "Minimum number of files that need to be visited before showing a 'slow enumeration' warning. Increase this value to reduce warnings in large workspaces.",
+                    "scope": "resource"
+                },
```

Comment:
hmm i'd prefer to minimize the number of new settings we add just to keep the config as simple as possible (partially because i don't like how messy the config system is, see #64). in your workspace did you have to change both settings to get the error to stop working? if not, i think we should probably just stick with the one `basedpyright.analysis.fileEnumerationTimeout` setting

Reply:
💯 we don't need that -- thanks for spotting it!

Yeah, I was going through the config system and totally agree. Would love to streamline it down the line. I could look at that down the line (but no guarantees at this stage).